otx/cli/tools/train.py

```diff
@@ -170,7 +170,7 @@ def main():
     validation_dataset = dataset.get_subset(Subset.VALIDATION)
     predicted_validation_dataset = task.infer(
         validation_dataset.with_empty_annotations(),
-        InferenceParameters(is_evaluation=True),
+        InferenceParameters(is_evaluation=False),
```

Comment:
Doesn't this affect the scores of other tasks? If this is True, the current action is not working, right?

Reply:
If is_evaluation is True, then task evaluates dataset with empty annotation. Therefore the result always be 0